src/SQLQueryStress/Form1.cs

```diff
@@ -523,35 +522,24 @@ private void ExportBenchMarkToTextFile(string fileName)
             {
                 MessageBox
                     .Show("Error While Saving BenchMark",
-                    string.Format("There was an error saving the benchmark to '{0}', make sure you have write privileges to that path", fileName));
+                    $"There was an error saving the benchmark to '{fileName}', make sure you have write privileges to that path");
             }
         }
 
         private void WriteBenchmarkTextContent(TextWriter tw)
         {
-            tw.WriteLine(string.Format("Test ID: {0}",
-                                _testGuid));
-            tw.WriteLine(string.Format("Test TimeStamp: {0}",
-                                _testStartTime));
-            tw.WriteLine(string.Format("Elapsed Time: {0}",
-                elapsedTime_textBox.Text));
-            tw.WriteLine(string.Format("Number of Iterations: {0}",
-                (int)iterations_numericUpDown.Value));
-            tw.WriteLine(string.Format("Number of Threads: {0}",
-                (int)threads_numericUpDown.Value));
-            tw.WriteLine(string.Format("Delay Between Queries (ms): {0}",
-                int.Parse(queryDelay_textBox.Text)));
-            tw.WriteLine(string.Format("CPU Seconds/Iteration (Avg): {0}",
-                cpuTime_textBox.Text));
-            tw.WriteLine(string.Format("Actual Seconds/Iteration (Avg): {0}",
-                actualSeconds_textBox.Text));
-            tw.WriteLine(string.Format("Iterations Completed: {0}",
-                iterationsSecond_textBox.Text));
-            tw.WriteLine(string.Format("Client Seconds/Iteration (Avg): {0}",
-                avgSeconds_textBox.Text));
-            tw.WriteLine(string.Format("Logical Reads/Iteration (Avg): {0}",
-                logicalReads_textBox.Text));
-            tw.WriteLine("");
+            tw.WriteLine($"Test ID: {_testGuid}");
+            tw.WriteLine($"Test TimeStamp: {_testStartTime}");
+            tw.WriteLine($"Elapsed Time: {elapsedTime_textBox.Text}");
+            tw.WriteLine($"Number of Iterations: {(int)iterations_numericUpDown.Value}");
+            tw.WriteLine($"Number of Threads: {(int)threads_numericUpDown.Value}");
+            tw.WriteLine($"Delay Between Queries (ms): {int.Parse(queryDelay_textBox.Text, CultureInfo.CurrentCulture)}");

```

Comment:
Use Invariant - for example to avoid https://stackoverflow.com/questions/2720024/int-parse-of-8-fails-int-parse-always-requires-cultureinfo-invariantculture